src/Mod/PartDesign/WizardShaft/WizardShaftTable.py

```diff
@@ -40,15 +40,15 @@ class WizardShaftTable:
     }
     rowDictReverse = {}
     headers = [
-                "Length [mm]",
-               "Diameter [mm]",
-               "Inner diameter [mm]",
-               "Constraint type",
-               "Start edge type",
-               "Start edge size",
-               "End edge type",
-               "End edge size"
-              ]    
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "Length [mm]"),
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "Diameter [mm]"),
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "Inner diameter [mm]"),
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "Constraint type"),
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "Start edge type"),
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "Start edge size"),
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "End edge type"),
+               "QT_TRANSLATE_NOOP("WizardShaftTable", "End edge size)"
+              ]
```

Comment:
Likely quotation mark discrepancy

Reply:
Missing closing double quotes after closing parenthesis, right?

Reply:
I was thinking more on the lines of removing quotation before `QT_TRANSLATE_NOOP`.
Plus at line 50 there should be a quote end before the parenthesis.

Reply:
Fixed in c114a7a  ?

Reply:
Perhaps an `import`?

Reply:
Fixed by using `QtCore.QT_...`